code/modules/mapping/station_datums.dm

```diff
@@ -23,6 +23,12 @@
 	webmap_url = "https://webmap.affectedarc07.co.uk/maps/paradise/cerestation/"
 	min_players_random = 60
 
+/datum/map/EmeraldStation
+	fluff_name = "NSS Seraphim"
+	technical_name = "EmeraldStation"
+	map_path = "_maps/map_files/stations/emeraldstation.dmm"
+	webmap_url = "https://www.youtube.com/watch?v=dQw4w9WgXcQ&pp=ygUIcmlja3JvbGw%3D"
```

Comment:
>:(

Reply:
i GENUINELY forgot I did that because I didnt have a webmap page yet... Ill talk to AA lol

Reply:
Webmap can be done when its ingame, I cant render it till then unless I do funky shit, which we will prolly need for TM memes

Reply:
new webmap link provided